sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
@daniel-k 
> I could imagine to implement AUTOCCA for the at86rf2xx by setting minBE = maxBE = 0 and CSMA_RETRIES = 0. 

Why would you set `minBE = maxBE = 0` for CCA? In my understanding just checks if the medium is currently free for sending. If this is not the case, the transceiver initiates another channel sensing according to the configured backoff time. So why would you fix this backoff time if CCA is enabled and why would you disable retransmissions in this case?